src/app/routes/getInitialData/getBaseUrl.test.js

```diff
@@ -0,0 +1,26 @@
+import getBaseUrl from './getBaseUrl';
+
+describe('Client', () => {
+  const logicMap = [
+    ['https://www.bbc.com', 'https://www.bbc.com'],
+    ['https://www.bbc.co.uk', 'https://www.bbc.co.uk'],
+    ['https://simorgh.api.bbci.co.uk', 'https://www.bbc.co.uk'],
+    ['https://www.beethoven.co.uk', 'https://www.bbc.co.uk'],
+    ['https://www.stage.bbc.com', 'https://www.stage.bbc.com'],
+    ['https://www.stage.bbc.co.uk', 'https://www.stage.bbc.co.uk'],
+    ['https://www.test.bbc.com', 'https://www.test.bbc.com'],
+    ['https://www.test.bbc.co.uk', 'https://www.test.bbc.co.uk'],
+    ['https://simorgh.test.api.bbci.co.uk', 'https://www.test.bbc.co.uk'],
+    ['https://www.beethoven.test.co.uk', 'https://www.test.bbc.co.uk'],
```

Comment:
Do we _want_ this behaviour or is it just a side effect of how we've written it?

Tbh I'd expect any non-BBC domain to default to bbc.com for the data.

Reply:
If we do this we need to open up the cors for alot of endpoints. also we want to ensure that test simorgh gets test data, even when not via test.bbc.com

Reply:
> If we do this we need to open up the cors for alot of endpoints

whether `https://www.beethoven.test.co.uk` requests data over https://www.test.bbc.co.uk or https://www.bbc.com, it's going to get a CORs error, right...?

Reply:
Did you intend `https://www.beethoven.test.co.uk` to be `https://www.beethoven.test.bbc.co.uk` instead?

Reply:
Chatted on slack, agreed to change url